tutorials/scripting/gdextension/gdextension_file.rst

```diff
@@ -0,0 +1,136 @@
+| **web**                       | Web browser                                                                                          |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+
+Build
+^^^^^
+
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| Flag                          | Description                                                                                          |
++===============================+======================================================================================================+
+| **debug**                     | Build with debug symbols                                                                             |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **release**                   | Optimized build without debug symbols                                                                |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **editor**                    | Editor build                                                                                         |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+
+Architecture
+^^^^^^^^^^^^
+
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| Flag                          | Description                                                                                          |
++===============================+======================================================================================================+
+| **double**                    | double-precision build                                                                               |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **single**                    | single-precision build                                                                               |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **64**                        | 64-bit build (any architecture)                                                                      |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **32**                        | 32-bit build (any architecture)                                                                      |
```

Comment:
Given that, for example, x86_64 and arm64 are not binary-compatible, I don't think it's worth mentioning/encouraging `64` or `32` as options. I think we should link to Godot's feature tag documentation for a complete list, and on this documentation page only list the ones we encourage using for GDExtension. Same with `x86` and `arm`, those should probably not be used with GDExtension, instead `arm64` etc should be used.

Also, Godot doesn't support 64-bit WebAssembly yet.

Reply:
I removed the feature tags that should not be advertised as prominently as suggested